scalableminds/webknossos
# Show organization in dataset info tab

### Steps to test:
- You need to set up an additional organization locally. The easiest way to do this is to set isWkorgInstance and mail.logToStdout to true each in the application.conf. Then, log out and create a new account. An confirmation link should be printed in the console. Click the link and then finish the registration. Then, the second account will belong to a separate organization.
Then, switch back to the first account (or do this beforehand) and make a dataset public via the dataset settings. Then, open the dataset url with the second account so that the base condition is met. (credits to @philippotto for this description)
- Check this in three places: 
-- In the dataset list view, after clicking on a dataset.
-- When viewing the dataset in the dataset info tab on the right.
-- When viewing or creating an annotation, again in the dataset info tab.
- (If the setup seems to hard, you can also change the checks === to !== so that you can test it with a dataset that belongs to your organization.)

### Issues:
- fixes #6922

------
(Please delete unneeded items, merge only when none are left open)
- [x] ~~Updated [documentation](../blob/master/docs) if applicable~~ doesnt seem applicable
- [x] Considered [common edge cases](../blob/master/.github/common_edge_cases.md). note: also works if activeUser is null, so if no one is logged in
